src/config/Config.zig

```diff
@@ -610,6 +610,10 @@ palette: Palette = .{},
 /// Specified as either hex (`#RRGGBB` or `RRGGBB`) or a named X11 color.
 @"unfocused-split-fill": ?Color = null,
 
+/// The color of the separator between splits. If unset the default system color
+/// will be used. GTK only.
+@"split-separator-color": ?Color = null,
```

Comment:
GTK may use this terminology but our internals have always used "divider" terminology. So that's why I chose that.